src/lib389/cli/dscontainer

```diff
@@ -58,6 +59,9 @@ from lib389.idm.directorymanager import DirectoryManager
 # is always available!
 log = setup_script_logger("container-init", True)
 
+# PID FILE
+PID_FILE=/data/run/slapd-localhost.pid
```

Comment:
lib389 has Paths module that returns constants like this. This module is already used by `dscontainer` for getting DS version. To get PID file location you can use something like this:
```
paths = Paths(serverid='localhost')
PID_FILE = paths.pid_file
```
And then you can also use `pid_from_file` from lib389 to read the file.

Reply:
Great.  Will update.  As mentioned here https://github.com/389ds/389-ds-base/discussions/6058#discussioncomment-8326466 some developer API docs would be good (or a pointer to them if I'm overlooking them).  Combing through all Python files trying to understand the API isn't great for contributors.

Reply:
We used to have lib389.readthedocs.io, but now it seems absent. @droideck, do you know why we don't have it anymore?

Reply:
I'm not sure the `paths.pid_file` approach is going to work since it appears dscontainer actually avoids the typical slapd start and uses subprocess command instead (supplying pidfile there): https://github.com/slominskir/389-ds-base/blob/65a113e9cdff8af918032a0d21beb04a78852461/src/lib389/cli/dscontainer#L338-L341

Even if `paths.pid_file` did have the correct value, it appears unnecessary to look it up as it's assigned to subprocess command in dscontainer so we already know it because we assigned it in dscontainer.